presto-main/src/main/java/com/facebook/presto/type/VarcharOperators.java

```diff
@@ -238,13 +238,13 @@ public static long hashCode(@SqlType("varchar(x)") Slice value)
         return xxHash64(value);
     }
 
-    @LiteralParameters({"x", "y"})
+    @LiteralParameters("x")
     @ScalarOperator(IS_DISTINCT_FROM)
     @SqlType(StandardTypes.BOOLEAN)
     public static boolean isDistinctFrom(
             @SqlType("varchar(x)") Slice left,
             @IsNull boolean leftNull,
-            @SqlType("varchar(y)") Slice right,
```

Comment:
The original function signature looked like a legit attempt to avoid coercion. What was wrong with that?